app-drawer/app-drawer.html

```diff
@@ -314,9 +313,7 @@
        * @method resetLayout
        */
       resetLayout: function() {
-        this.debounce('_resetLayout', function() {
-          this.fire('app-drawer-reset-layout');
-        }, 1);
+        this.fire('app-drawer-reset-layout');
```

Comment:
This seems good since now both `app-header` and `app-drawer` fires the `*-reset-layout` event sync: https://github.com/PolymerElements/app-layout/blob/master/app-header/app-header.html#L485